js/desktopCapturer/index.js

```diff
@@ -0,0 +1,140 @@
+'use strict';
+
+const electron = require('electron');
+const BrowserWindow = electron.BrowserWindow;
+const ipc = electron.ipcMain;
+const path = require('path');
+const fs = require('fs');
+const log = require('../log.js');
+const logLevels = require('../enums/logLevels.js');
+const { isMac, isWindowsOS } = require('./../utils/misc.js');
+
+let screenPickerWindow;
+let preloadWindow;
+let eventId;
+
+let windowConfig = {
+    width: 580,
+    height: isMac ? 519 : 523,
+    show: false,
+    modal: true,
+    frame: false,
+    autoHideMenuBar: true,
+    resizable: false,
+    alwaysOnTop: true,
+    webPreferences: {
+        preload: path.join(__dirname, 'renderer.js'),
+        sandbox: true,
```

Comment:
is this needed?

Reply:
Yes, It makes this `browserWindow` more secure 😄